Octokit/Models/Response/ResourceRateLimit.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Diagnostics;
+using System.Globalization;
+
+using Octokit.Helpers;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class ResourceRateLimit
```

Comment:
Three models in use:

1. Originally MiscRateLimit -> Changed to MiscellaneousRateLimit to avoid abbreviation
2. Originally ResourceRateLimits -> Changes to ResourceRateLimit 
3. Originally ResourceRateLimit -> Changes to use existing RateLimit

Hopefully that makes sense